frontend/src/ts/utils.ts

```diff
@@ -537,3 +537,23 @@ export const isJson = (str) => {
   }
   return true;
 };
+
+export const createDownloadLink = (url: string, name: string) => {
+  const link = document.createElement('a');
+  link.href = url;
+  link.download = name;
+  link.click();
+};
+
+export const getDateTimeWithTimezoneString = (
+  timestamp: number,
+  timezone: string
+) => {
+  return moment
```

Comment:
Move this method to `utils`? and then use `formatDate(time)`

Reply:
I will move those two functions to base-lib. However, I think it's unnecessary to use `formatDate()`. First, `formatDate()` is taking `Date` type parameters while those two functions are taking number type timestamp. Second, I'd rather use *moment-timezone* lib build-in `format()`.